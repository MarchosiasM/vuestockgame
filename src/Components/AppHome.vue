<template>
  <div class="container">
    <h1>Trade or View Your Portfolio</h1>
    <p>You may Save & Load your Data</p>
    <p>Click on 'End Day' to begin a new Day!</p>
    <hr>
    <h2>Your Funds: {{ funds }}</h2>
  </div>
</template>

<script>
export default {
  data() {
    return {
      funds: 10000
    };
  }
};
</script>
