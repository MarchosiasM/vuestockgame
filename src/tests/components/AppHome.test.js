import { mount } from '@vue/test-utils';
import AppHome from '../../Components/AppHome.vue';

describe('Home element', () => {
  it('renders correctly', () => {
    const wrapper = mount(AppHome);
    expect(wrapper.element).toMatchSnapshot();
  });
});
