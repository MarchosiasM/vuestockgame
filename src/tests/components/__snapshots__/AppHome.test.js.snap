// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home element renders correctly 1`] = `
<div
  class="container"
>
  <h1>
    Trade or View Your Portfolio
  </h1>
   
  <p>
    You may Save & Load your Data
  </p>
   
  <p>
    Click on 'End Day' to begin a new Day!
  </p>
   
  <hr />
   
  <h2>
    Your Funds: 10000
  </h2>
</div>
`;
